backend/internal/service/handler/review.go

```diff
@@ -0,0 +1,104 @@
+package handler
+
+import (
+	"fmt"
+	"platnm/internal/storage"
+	"strconv"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+type ReviewHandler struct {
+	reviewRepository storage.ReviewRepository
+}
+
+func NewReviewHandler(reviewRepository storage.ReviewRepository) *ReviewHandler {
+	return &ReviewHandler{
+		reviewRepository,
+	}
+}
+
+func (h *ReviewHandler) GetReviews(c *fiber.Ctx) error {
+	review, err := h.reviewRepository.GetReviews(c.Context())
+	if err != nil {
+		return err
+	}
+
+	return c.Status(fiber.StatusOK).JSON(review)
+}
+
+func (h *ReviewHandler) GetReviewById(c *fiber.Ctx, mediaType string) error {
+	id := c.Params("id")
+	offsetstr := c.Query("offset", "0")
```

Comment:
`fiber.Ctx` has a nice helper for this, `func (c *Ctx) QueryInt(key string, defaultValue ...int) int`